File: app/src/p_widget/main/java/com/llj/widget/ui/activity/ConstraintActivity.java
package com.llj.widget.ui.activity;

import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;
import com.llj.architecturedemo.MyBaseActivity;
import com.llj.architecturedemo.R;
import com.llj.component.service.arouter.CRouter;

/**
 * ArchitectureDemo
 * describe:
 * author liulj
 * date 2018/7/5
 */
@Route(path = CRouter.WIDGET_CONSTRAINT_ACTIVITY)
public class ConstraintActivity extends MyBaseActivity {
    @Override
    public int layoutId() {
        return R.layout.activity_constraint;
    }

    @Override
    public void initViews(Bundle savedInstanceState) {

    }

    @Override
    public void initData() {

    }
}

File: app/src/p_widget/main/java/com/llj/widget/ui/activity/ConstraintActivity.kt
package com.llj.widget.ui.activity

import android.os.Bundle
import androidx.viewbinding.ViewBinding
import com.alibaba.android.arouter.facade.annotation.Route
import com.llj.application.router.CRouter
import com.llj.architecturedemo.MainMvcBaseActivity
import com.llj.architecturedemo.R

/**
 * ArchitectureDemo
 * describe:
 *
 * @author llj
 * @date 2018/7/5
 */
@Route(path = CRouter.APP_CONSTRAINT_ACTIVITY)
class ConstraintActivity : MainMvcBaseActivity<ViewBinding>() {


  override fun setContentView(layoutResID: Int) {
    super.setContentView(layoutResID)
  }

  override fun layoutId(): Int {
    return R.layout.activity_constraint
  }

  override fun initViews(savedInstanceState: Bundle?) {
  }

  override fun initData() {

  }
}
